### ISSUE TITLE
Email button

### ISSUE THREAD

**Issue by thogebati** [amosproj/amos2023ws01-ticket-chat-ai#14]
## User story
1. As a that creates a ticket via the WebApp
2. I want / need a button to create a preconfigured email
3. So that I only need to input my problem

## Acceptance criteria
* After clicking the button my email program opens, with the mail open
* The mail address, subject and body are filled in correctly

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
